lib/ui/index.ts

```diff
@@ -54,7 +54,10 @@ export function uiLink(linkText: string, url: string): string {
     : `${linkText}: ${encodedUrl}`;
 }
 
-export function uiAccountDescription(accountId: number, bold = true): string {
+export function uiAccountDescription(
+  accountId: number | undefined,
```

Comment:
`accountId?: number`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1288/commits/dbfad7214b180917cc226ffaee59f96b198bf5f3